### DISCUSSION TITLE
Alternatives to Boost mpi

### DISCUSSION THREAD

**Discussion by vincentchabannes** [feelpp/feelpp#2247] [Ideas]
- MPL : A message passing library. Links [github](https://github.com/rabauke/mpl) and [doc](https://rabauke.github.io/mpl/html/)
 - KaMPIng: Karlsruhe MPI next generation. Links [github](https://github.com/kamping-site/kamping) and [doc](https://kamping-site.github.io/kamping/)
 - MPI : Modern C++20 message passing interface wrapper. Links [github](https://github.com/acdemiralp/mpi)

see also this [paper](https://vr.rwth-aachen.de/media/papers/243/A_C20_Interface_for_MPI_4.0.pdf)

**Comment by prudhomm** [comment]
There is now https://gitlab.com/correaa/boost-mpi3 a MPI3 boost mpi replacement
